pkg/cli/tools/binary_tools.go

```diff
@@ -17,21 +17,17 @@ limitations under the License.
 package tools
 
 import (
-	"context"
 	"fmt"
+	"io"
+	"net/http"
```

Comment:
Does it make sense to rename this file? It's called `binary_tools.go` because it used to handle multiple tools (there used to be multiple things we downloaded). Should this go the bicep package and be renamed?

Reply:
That makes sense. Move to the bicep folder and renamed to `download_tools`